app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,20 @@
-<h2>Forgot your password?</h2>
+<div class="px-[20vw] py-8 flex flex-col items-center gap-4 selection:bg-blue-500 selection:text-white">
+  <h2 class="text-blue-400 text-2xl sm:text-4xl font-semibold mb-8">Forgot your password?</h2>
 
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="bg-blue-400 text-white rounded-md p-8 shadow-md">
+    <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+      <div class="field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "bg-white text-blue-500 rounded-md p-4 text-sm w-full" %>
```

Comment:
Do we use auto sort ?
This https://tailwindcss.com/blog/automatic-class-sorting-with-prettier
should be added to linter as well https://github.com/trkin/kindergarten-exchange/issues/11